src/llm_vm/onsite_llm.py

```diff
@@ -184,6 +190,190 @@ def asynctune():
     def finetune_immediately(self):
         finetune()()
 
+
+class TokenStreamerAsStoppingCriterion:
+    def __init__(self, token_streamer):
+        self.token_streamer = token_streamer
+
+    def __call__(self, input_ids, scores, **kwargs):
+        if self.token_streamer is None:
+            return False
```

Comment:
I think itd be appropriate to return None here

Reply:
I agree